Traceback (most recent call last):
  File "parser.py", line 50, in <module>
    ast = parser.parse(text, lexer,debug=0)
  File "/usr/local/lib/python2.7/dist-packages/ply/yacc.py", line 331, in parse
    return self.parseopt_notrack(input, lexer, debug, tracking, tokenfunc)
  File "/usr/local/lib/python2.7/dist-packages/ply/yacc.py", line 1106, in parseopt_notrack
    p.callable(pslice)
  File "/home/nico/Desktop/tLeng2016(posta)/tleng2016/parser_rules.py", line 468, in p_vecVal2
    chequearAccesoVector(subexpressions)
  File "/home/nico/Desktop/tLeng2016(posta)/tleng2016/parser_rules.py", line 1198, in chequearAccesoVector
    nombreVar1 = subexpressions[1]["var"]
KeyError: 'var'
Traceback (most recent call last):
  File "parser.py", line 50, in <module>
    ast = parser.parse(text, lexer,debug=0)
  File "/usr/local/lib/python2.7/dist-packages/ply/yacc.py", line 331, in parse
    return self.parseopt_notrack(input, lexer, debug, tracking, tokenfunc)
  File "/usr/local/lib/python2.7/dist-packages/ply/yacc.py", line 1106, in parseopt_notrack
    p.callable(pslice)
  File "/home/nico/Desktop/tLeng2016(posta)/tleng2016/parser_rules.py", line 1026, in p_tCompare
    chequearUnicoTerminal(subexpressions, ["int", "float"])
  File "/home/nico/Desktop/tLeng2016(posta)/tleng2016/parser_rules.py", line 1194, in chequearUnicoTerminal
    chequearTipo(subexps, tipos)
  File "/home/nico/Desktop/tLeng2016(posta)/tleng2016/parser_rules.py", line 1113, in chequearTipo
    raise Exception(message)
Exception: Se esperaba tipo int o float
Traceback (most recent call last):
  File "parser.py", line 50, in <module>
    ast = parser.parse(text, lexer,debug=0)
  File "/usr/local/lib/python2.7/dist-packages/ply/yacc.py", line 331, in parse
    return self.parseopt_notrack(input, lexer, debug, tracking, tokenfunc)
  File "/usr/local/lib/python2.7/dist-packages/ply/yacc.py", line 1106, in parseopt_notrack
    p.callable(pslice)
  File "/home/nico/Desktop/tLeng2016(posta)/tleng2016/parser_rules.py", line 449, in p_vecVal
    chequearAccesoVector(subexpressions)
  File "/home/nico/Desktop/tLeng2016(posta)/tleng2016/parser_rules.py", line 1211, in chequearAccesoVector
    raise Exception("El indice de un vector solo se puede ser de tipo int") 
Exception: El indice de un vector solo se puede ser de tipo int
Traceback (most recent call last):
  File "parser.py", line 50, in <module>
    ast = parser.parse(text, lexer,debug=0)
  File "/usr/local/lib/python2.7/dist-packages/ply/yacc.py", line 331, in parse
    return self.parseopt_notrack(input, lexer, debug, tracking, tokenfunc)
  File "/usr/local/lib/python2.7/dist-packages/ply/yacc.py", line 1106, in parseopt_notrack
    p.callable(pslice)
  File "/home/nico/Desktop/tLeng2016(posta)/tleng2016/parser_rules.py", line 371, in p_funcString
    chequearTipo([subexpressions[3]], ["string"])
  File "/home/nico/Desktop/tLeng2016(posta)/tleng2016/parser_rules.py", line 1113, in chequearTipo
    raise Exception(message)
Exception: Se esperaba tipo string
